### ISSUE TITLE
Naming issue

### ISSUE THREAD

**Issue by K4N0** [websitenmaker12/Advanced-Genetics#164]
advancedgenetics-1.7.10-1.5.9
I have just installed this mod and tried it out.
I get a syringe, draw my blood and certrifuge it.
I create a Gene eat grass [completed]
I combine them in the DNA combiner and instead of getting a 
*_Syringe filled eat grass, I just get a Syringe filled.
I then centrifuge it again and instead of getting a
*_Syringe filled eat grass [centrifuged], I get a syringe filled [centrifuged]
If i inject this i get the effects and everything works well, but if i didn't know or thought it didn't work. I can keep trying over and over again endless believing it didn't work.
![2015-06-19_22 02 01](https://cloud.githubusercontent.com/assets/10932905/8266092/08e526ce-16cf-11e5-8c78-a1107cbf5fac.png)
![2015-06-19_22 02 10](https://cloud.githubusercontent.com/assets/10932905/8266093/0d63d1a0-16cf-11e5-91a2-90024da88650.png)
![2015-06-19_22 02 12](https://cloud.githubusercontent.com/assets/10932905/8266094/11c9bd22-16cf-11e5-9762-53a092f099c7.png)
![2015-06-19_22 02 17](https://cloud.githubusercontent.com/assets/10932905/8266095/1539f95e-16cf-11e5-8321-742dc2033f84.png)

**Comment by K4N0** [comment]
This is after i use the dna combiner. If i inject this i will get the effect, but the naming just isn't there. I'll try removing nei or other to see if that is causing this.
![2015-06-19_22 04 47](https://cloud.githubusercontent.com/assets/10932905/8266101/45eb8f0e-16cf-11e5-874e-686a65e3e43e.png)

**Comment by tim-oster** [comment]
This isn't a bug. It just don't show which genes are in the syringe. Therfore, you can use the Microscope.

**Comment by K4N0** [comment]
is there a way to stop it from letting you loop it over and over again. I was able to add same and other genes to the same syringe, causing me to waste products. I saw a mod showcase and it showed the gene in the syringe which is why i thought it might be a bug.
I'm just saying that we don't use a microscope to see what kind of gene we are holding, so why do that with the syringe? showing what kind of gene is in the syringe is probably the best way to fix this issue. unless you have other plans for the way things will work in the future.
